### DISCUSSION TITLE
Portuguese language

### DISCUSSION THREAD

**Discussion by cschlindwein** [amidaware/tacticalrmm#1691] [Ideas]
How can I help with Portuguese language translation

**Comment by silversword411** [comment]
Sorry, that's currently lower on the priority list. 

We do welcome any internationalization PRs. Someone tried a French one but went around it in the wrong way. See https://github.com/amidaware/tacticalrmm/issues/39#issuecomment-673613776

Also https://github.com/amidaware/tacticalrmm/issues/39
